Added AutoAI for credit risk

jrtorres opened the pull request:
AutoAI branch with credit risk data.

scottdangelo commented on workshop/machine-learning-autoai/README.md:
```diff
@@ -0,0 +1,275 @@
+
+* From the project overview page, click on the `Assets` tab to open the assets page where your project assets are stored and organized. You can load a local dataset using the interface on the right.
+
+* Click `browse` to find and upload the [german-credit-data.csv](../dataset/Telco-Customer-Churn.csv) dataset from your machine:
```
This link should be `https://github.com/IBM/credit-risk-workshop-cpd/blob/master/data/original/german_credit_data.csv` for the online link.
or could be `../../data/german_credit_data.csv` for local.

jrtorres commented:
Fixed the local link for csv file in auto ai instructions. I left the online link to the raw link, so users can be instructed to just "right click -> save as" to download the csv file. Also modified the table of contents summary file for name of autoai section to be consistent with the jupyter approach.
